readme.md

```diff
@@ -54,30 +68,213 @@ Available environment variables:
 | HTTP_HOST | `0.0.0.0`     |                                                                                                           |
 | HTTP_PORT | `3000`        |                                                                                                           |
 
+### Routes
+
+| Method | Path                     | description                                                                         |
+| ------ | ------------------------ | ----------------------------------------------------------------------------------- |
+| POST   | /v1/decoder              | General decoder route. Need to specify [protocol](#protocols) in the requests body  |
+| POST   | /v1/decoder/analog       | Decoder for the `analog` protocol based devices                                     |
+| POST   | /v1/decoder/mtx          | Decoder for the `mtx` protocol based devices                                        |
+| POST   | /v1/decoder/mtxLora      | Decoder for the `mtxLora` protocol based devices                                    |
+| POST   | /v1/decoder/obisObserver | Decoder for the `obisObserver` protocol based devices                               |
+| POST   | /v1/encoder              | General encoder route. Need to specify [protocol](#protocols)  in the requests body |
+| POST   | /v1/encoder/analog       | Encoder for the `analog` protocol based devices                                     |
+| POST   | /v1/encoder/mtx          | Encoder for the `mtx` protocol based devices                                        |
+| POST   | /v1/encoder/mtxLora      | Encoder for the `mtxLora` protocol based devices                                    |
+| POST   | /v1/encoder/obisObserver | Encoder for the `obisObserver` protocol based devices                               |
+
+
+### POST request parameters
+
+#### Framing format
+
+| name | value | description       |
+| ---- | ----- | ----------------- |
+| NONE | `0`   | no framing        |
+| HDLC | `1`   | HDLC frame format |
+
+Default value: `0`
+Example: `framingFormat: 1`
+
+#### Bytes conversion format
+
+| name   | value | description                   |
+| ------ | ----- | ----------------------------- |
+| HEX    | `1`   | data treats as hex string     |
```

Comment:
```suggestion
| `HEX`    | `1`   | data treats as hex string     |
```
each number/term/token/env var and so on would be nice to highlight

Reply:
fixed